molgenis-semantic-mapper/src/main/java/org/molgenis/data/mapper/exception/DuplicateMappingProjectException.java

```diff
@@ -0,0 +1,35 @@
+package org.molgenis.data.mapper.exception;
+
+import static java.lang.String.format;
+
+// S2166 'Classes named like "Exception" should extend "Exception" or a subclass' often gives false positives at dev time
+@SuppressWarnings({ "squid:MaximumInheritanceDepth", "squid:S2166" })
+public class DuplicateMappingProjectException extends MappingServiceException
```

Comment:
maybe replace occurrences EntityTypeAlreadyExistsException

Reply:
assign

Reply:
proposal: keep exception, since the usage is weird because the entity might exist